docs/src/components/productX/XPlans.tsx

```diff
@@ -1,83 +1,63 @@
 import * as React from 'react';
-import { styled } from '@mui/material/styles';
-import Box from '@mui/material/Box';
+import Stack from '@mui/material/Stack';
 import Grid from '@mui/material/Grid';
 import Typography from '@mui/material/Typography';
-import Button from '@mui/material/Button';
-import KeyboardArrowRightRounded from '@mui/icons-material/KeyboardArrowRightRounded';
 import Section from 'docs/src/layouts/Section';
 import SectionHeadline from 'docs/src/components/typography/SectionHeadline';
 import GradientText from 'docs/src/components/typography/GradientText';
-import Link from 'docs/src/modules/components/Link';
 import ROUTES from 'docs/src/route';
-import Frame from 'docs/src/components/action/Frame';
+import InfoCard from 'docs/src/components/action/InfoCard';
+import IconImage from 'docs/src/components/icon/IconImage';
+import GetStartedButtons from 'docs/src/components/home/GetStartedButtons';
 
-const Image = styled('img')(({ theme }) => ({
-  filter: 'drop-shadow(-8px 4px 20px rgba(61, 71, 82, 0.1))',
-  transition: '0.3s',
-  display: 'block',
-  height: 'auto',
-  borderRadius: '8px',
-  ...theme.applyDarkStyles({
-    filter: 'drop-shadow(-8px 4px 20px rgba(0, 0, 0, 0.4))',
-  }),
-}));
+const content = [
+  {
+    icon: <IconImage name="pricing/x-plan-community" mode="" loading="eager" />,
+    title: 'Community version',
+    description:
+      'Free forever. This version contains features that we believe are maintainable by contributions from the open-source community.',
+  },
+  {
+    icon: <IconImage name="pricing/x-plan-pro" mode="" loading="eager" />,
+    title: 'Pro plan',
+    description:
+      'More advanced capabilities such as multi-filtering, multi-sorting, and more for the Data Grid; as well as the other components.',
```

Comment:
```suggestion
      'More advanced capabilities such as multi-filtering, multi-sorting, and more for the Data Grid, or range selection capabilities for Date and Time Pickers.',
```

Reply:
I didn't necessarily want to pin down just these two components, even if the case was the Pro plan only covering them at the moment. It's a step towards making MUI X mean more than the Data Grid 😬 Will wait for @samuelsycamore to chime in here, too!

Reply:
Does it make sense to write in such a way that we consider the other components, too (even if they may still technically not be part of the Pro plan yet—not if it's the case)?

Reply:
Yeah, that makes sense, too. Alright — let's resolve this for now. 😬